### ISSUE TITLE
Unable to adjust H3O or OH on My Solution with Jaws

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/ph-scale#340]
**Device**
Samsung Laptop
**OS**
Win 11
**Browser**
Chrome + Jaws
**Problem Description**
For https://github.com/phetsims/qa/issues/1292
I was unable to move the H3O or OH boxes on the My Solution screen when using Jaws. I tried both with the arrow keys and with WASD.

**Comment by jessegreenberg** [comment]
Confirmed with JAWS. I am guessing this will be fixed by using AccessibleDraggableOptions.

**Comment by jessegreenberg** [comment]
@marlitas I went ahead and committed this patch because I proceeded with https://github.com/phetsims/scenery/issues/1738 and so this case was now throwing an assertion.

Can you please verify and review?

**Comment by marlitas** [comment]
Oh I think we hit it at similar points? I just pushed as well. Looks good to me. @KatieWoe can you confirm this is now behaving as expected and close if so?

**Comment by KatieWoe** [comment]
Looks good on main
